modules/core/src/test/java/org/apache/ignite/testframework/GridTestUtils.java

```diff
@@ -140,7 +140,7 @@
     public static final long DFLT_BUSYWAIT_SLEEP_INTERVAL = 200;
 
     /** */
-    public static final long DFLT_TEST_TIMEOUT = 5 * 60 * 1000;
+    public static final long DFLT_TEST_TIMEOUT = 5000 * 60 * 1000;
```

Comment:
redefine timeout in certain test

Reply:
It wrong code. Sorry, I committed it by mistake.